.github/workflows/build.yml

```diff
@@ -0,0 +1,69 @@
+name: build
+
+on: push
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Node 12.7
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12.7
+      
+      - name: Cache Node modules
+        id: cache-node-modules
+        uses: actions/cache@v2
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
+
+      - name: Install dependencies
+        if: steps.cache-node-modules.outputs.cache-hit != 'true'
+        run: yarn install
+
+      - name: Check formatting
+        run: yarn check-formatting
+
+      - name: Lint
+        run: yarn lint
+  
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        node-version: [10.17, 12.7]
```

Comment:
Let's test Node version 14 here as well... we can do `10`, `12`, and `14`. If we leave out the minor version number, I believe we will just use the latest minor version within that major version, which is probably what we want here.